src/main/java/testProj/resources/UserResource.java

```diff
@@ -0,0 +1,27 @@
+package testProj.resources;
+
+import testProj.api.User;
+import testProj.core.UserService;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.Optional;
+
+@Path("/get-user")
+@Produces(MediaType.APPLICATION_JSON)
+public class UserResource {
+    @Inject
+    UserService userService;
+
+    public UserResource(UserService user) {
+        this.userService = user;
+    }
+
+    @GET
+    public Optional<User> getUser(){
```

Comment:
Could you please make this endpoint to return User by id sent as query parameter, like this: 
`/user?id=0d6948b9-a1d0-4c1c-83a2-d146a3b3848b`